fixture-monkey-kotlin/src/test/kotlin/com/navercorp/fixturemonkey/kotlin/test/PrimaryConstructorArbitraryIntrospectorTest.kt

```diff
@@ -63,6 +64,16 @@ class PrimaryConstructorArbitraryIntrospectorTest {
         then(actual).isNotEqualTo("default_value")
     }
 
+    @Property
+    fun sampleDuration() {
+        thenNoException().isThrownBy { sut.giveMeOne<Duration>() }
+    }
+
+    @Property
+    fun sampleDurationValue() {
+        thenNoException().isThrownBy { sut.giveMeOne<DurationValue>() }
+    }
```

Comment:
It is better to add examples to fix `Duration` and `DurationValue` for testing decomposing it.

Reply:
The test fails because we're decompressing the value. 
We've fixed it so that we can compare the values. 👍